websrc/source/documents/tutorials/prometheus-tutorial.md

```diff
@@ -0,0 +1,150 @@
+
+
+There are many other data monitoring platforms besides Prometheus and Grafana, such as Graphite, InfluxDB, OpenTSDB, Nagios, and Sensu. Because Prometheus and Grafana are open source platforms, they allow for easy integration and personalized configuration. To find out more about alternatives, click [here](https://prometheus.io/docs/introduction/comparison/).
+
+Let's explore Prometheus and Grafana.
+
+Note: These steps assume you are working with the demo environment and following along with the Kubernetes Networking and Policy tutorials.
+
+Prometheus and Grafana are running as nodeports which are exposed on the host IP. Since they are both configured to run on the master node, let's find out the IP of the master node.
+
+```
+[vagrant@kubeadm-master ~]$ kubectl describe nodes kubeadm-master
+```
+We can see that it's IP is 192.168.2.54.
+
+We can also find out what port Prometheus is exposed at.
+
+```
+[vagrant@kubeadm-master ~]$ kubectl get svc -n kube-system
+NAME         CLUSTER-IP       EXTERNAL-IP   PORT(S)          AGE
+grafana      10.100.180.79    <nodes>       3000:30001/TCP   1h
+kube-dns     10.96.0.10       <none>        53/UDP,53/TCP    1h
+prometheus   10.111.247.164   <nodes>       9090:30000/TCP   1h
+```
+Let's open a browser and navigate to http://192.168.2.54:30000. This will open up the Prometheus UI.
+
+Let's check and make sure that Prometheus is able to fetch data from all the nodes. At the top click on Status and in the drop down menu click on Targets. You should see that there are two targets and that they are `UP` and green. If it says `UNKNOWN` it is likely because the environment or prometheus is still configuring. 
+
+Go back to Graph from the top tool bar. Click on the drop down menu that says -insert metric at cursor- and pick any metric. Then click on execute and graph right below the execute button.
+
+Now let's move to Grafana. 
+
+Grafana is exposed at port 30001 so let's navigate to http://192.168.2.54:30001 in our browser. The Grafana UI will open up and prompt you for a username and password. Username is "admin" and password is "admin". We will be working with Grafana for the remainder of this tutorial because it is feature-rich, easy to use, and very flexible.
+
+After you have logged in, you will see an option to Add data source. Click on that. Fill out the information as show in the following screenshot.
+
+![image](link)
+
+Click on the Grafana logo in the top left of the screen and go to Dashboard and click on new.
+Click on graph. Click on "Panel Title" and then click on "Edit".
+
+### <a name="ch2"></a> Chapter 2: Examples
+
+#### Multi-host networking
+
+Now let's go back to our Kubernetes Networking Tutorial and complete chatper 2. 
+
+Instead of `ping -c 3 10.1.2.3` it maybe more telling to run `ping -c 1 10.1.2.3 -s 5000`. This tutorial has run this command three times (once every 15 seconds).
+
+We can add the following four queries:
```

Comment:
Add a statement to indicate the user has to go back to the Grafana dashboard at this point.